### DISCUSSION TITLE
x-reactions似乎没有完全同步状态？

### DISCUSSION THREAD

**Discussion by fushizhe** [alibaba/formily#2228] [Q&A 2.x] [answered]
https://codesandbox.io/s/wonderful-kirch-nqn5e?file=/App.tsx

请教个x-reactions的问题哈。操作路径在codesandbox里写了。控制者1 控制 控制者2和受控者1是否显示，同时受控者1依赖控制者2是否有值来决定是否是disabled状态。

**Comment by janryWang** [comment]
提个bug吧

**Reply by fushizhe** [reply]
https://github.com/alibaba/formily/issues/2234
bug已提，如果也有遇到这个问题的同学，可以先把逻辑写到onFieldReact里面就能避免哈。

**Comment by janryWang** [comment, marked as answer]
已经修复
